lib/QubitFlatfileImport.class.php

```diff
@@ -2148,7 +2156,7 @@ public function createRightAndRelation($options)
    *
    * @return stdClass  result object
    */
-  public function fetchKeymapEntryBySourceAndTargetName($sourceId, $sourceName, $targetName)
+  public static function fetchKeymapEntryBySourceAndTargetName($sourceId, $sourceName, $targetName)
```

Comment:
OK so today I learned calling static methods via $this-> is actually a thing in PHP (which seems odd), but it sounds like it may be removed in the future so I still think it's worth switching it to QubitFlatfileImport::

"In PHP 7, calling non-static methods statically is deprecated, and will generate an E_DEPRECATED warning. Support for calling non-static methods statically may be removed in the future."